src/expr/src/interpret.rs

```diff
@@ -896,6 +801,75 @@ impl<'a> Interpreter for ColumnSpecs<'a> {
     }
 }
 
+/// An interpreter that returns whether or not a particular expression is "pushdownable".
```

Comment:
This is very useful, you might want to refer to the `Trace` docs in the module-level docs of this module.